garp-on-svi-action-pack/garp-on-svi/script.py

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) 2022 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the COPYING file.
+
+def string_to_list(string_to_convert):
+    numbers = []
+    segments = [segment.strip() for segment in string_to_convert.split(",")]
+    for segment in segments:
+        if "-" in segment:
+            for i in range(int(segment.split("-")[0]), int(segment.split("-")[1]) + 1):
+                numbers.append(i)
+        else:
+            numbers.append(int(segment))
+    return numbers
+
+
+switch = ctx.getDevice()
+vlan_ids = ctx.changeControl.args['VLAN IDs']
+ctx.alog("{}".format(ctx.alog(vlan_ids)))
+
+if vlan_ids is not None:
+    vlan_ids = string_to_list(vlan_ids)
+else:
+    ctx.alog("No VLAN IDs provided")
+
+for vlan_id in vlan_ids:
+    ctx.alog("Getting interface info on {} for Vlan {}".format(switch.ip, vlan_id))
+    ctx.alog("show ip interface vlan {} output: {}".format(
+        vlan_id,
+        ctx.runDeviceCmds(["show ip interface vlan {}".format(vlan_id)])
+        )
```

Comment:
There's a few errors regarding these that I'm seeing: `closing bracket does not match indentation of opening bracket's line`
Here and on ln 36, 46 and 56